### PR TITLE
bootstrap play upgrade and fixed failing tests

### PR DESCRIPTION
**Pull request by kuldeepwadhwa9**
# DDCE-StoryNumberHere

## Bug fix / New feature (delete as appropriate)

Please include a summary / description of the change and which issue it fixes.  Include any relevant user needs, context or links to other PRs related to this PR (eg. acceptance tests, environment config).

## PR Suggestions
- Have you looked at the JIRA story to make sure all Acceptance Criteria has been met?
- Have you done a visual check of the changes?
- Is there any still commented or unused code? Lingering printlns?
- Have things been implemented in a complicated way?
- Are the test still over the coverage threshold?
- Does the compiler throw a lot of warnings? 
- Have methods and tests been named clearly?
- Were there any changes in the config? Do changes need to be made in app-config-???
- Have you done a manual walkthrough?

## Checklist PR Raiser
 - [ ]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [ ]  I've included appropriate tests with any code I've added (Unit, Integration, Acceptance etc.)
 - [ ]  I've executed the acceptance test pack locally to ensure there are no functional regressions (Performance tests where relevant)
 - [ ]  I've squashed my commits - including the JIRA issue number in the commit message
 - [ ]  I've run a dependency check to ensure all dependencies are up to date
 
## Checklist PR Reviewer
 - [ ]  I've verified every effort was made to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [ ]  I've verified appropriate tests were included with any code added (Unit, Integration, Acceptance etc.)
 - [ ]  I've executed the acceptance test pack locally to ensure there are no functional regressions (Performance tests where relevant)
 - [ ]  I've verified commits were squashed and rebased - including the JIRA issue number in the commit message
 - [ ]  I've run a dependency check to ensure all dependencies are up to date

### CODE REVIEW

File: project/AppDependencies.scala [hunk 1]
```diff
@@ -2,15 +2,15 @@ import sbt._
...
-    "uk.gov.hmrc" %% "url-builder" % "3.5.0-play-27",
+    "uk.gov.hmrc" %% "play-partials" % "8.1.0-play-28",
+    "uk.gov.hmrc" %% "url-builder" % "3.5.0-play-28",
     "uk.gov.hmrc" %% "tax-year" % "1.3.0",
-    "uk.gov.hmrc" %% "bootstrap-frontend-play-27" % "5.3.0",
-    "uk.gov.hmrc" %% "play-frontend-hmrc" % "0.68.0-play-27",
-    "uk.gov.hmrc" %% "http-caching-client" % "9.4.0-play-27",
+    "uk.gov.hmrc" %% "bootstrap-frontend-play-28" % "5.4.0",
+    "uk.gov.hmrc" %% "play-frontend-hmrc" % "0.81.0-play-28",
+    "uk.gov.hmrc" %% "http-caching-client" % "9.5.0-play-27",
```
Comment (anburoach) [inline project/AppDependencies.scala:13]: upgrade **http-caching-client** to play-28

File: project/AppDependencies.scala [hunk 2]
```diff
@@ -2,15 +2,15 @@ import sbt._
...
     compilerPlugin("com.github.ghik" % "silencer-plugin" % silencerVersion cross CrossVersion.full),
     "com.github.ghik" % "silencer-lib" % silencerVersion % Provided cross CrossVersion.full
   )
```
Comment (anburoach) [inline project/AppDependencies.scala:16]: Upgrade scalatestplus-play to 5.1.0

**Review by anburoach — CHANGES_REQUESTED** [review]
Remove SbtAutoBuildPlugin, SbtGitVersioning in build.sbt

File: project/AppDependencies.scala (hunk 1 above)
Comment (kuldeepwadhwa9) [reply project/AppDependencies.scala]: Done

**Review by anburoach — APPROVED** [review]
Approved

**Pull request merged**